kumascript/tests/macros/apiref.test.js

```diff
@@ -518,6 +518,7 @@ describeMacro("APIRef", function () {
   beforeEachMacro(function (macro) {
     // Mock calls to MDN.subpagesExpand
     macro.ctx.page.subpagesExpand = jest.fn((page) => {
+      // eslint-disable-next-line jest/no-standalone-expect
```

Comment:
Isn't it better to actually solve it. Perhaps something like `expect.hasAssertions(1);`

Reply:
If this is supposed to be a kind of test wrapper, it might be added to the `additionalTestBlockFunctions: ["itMacro", "withDeveloping"]`